docs/design-proposals/global-scheduler/GlobalSchedulerArchitectureDesignSpecificationFirstDraft.md

```diff
@@ -0,0 +1,416 @@
+    - [2.5.2 Multi-Dimensional Optimization Based Scheduling Algorithm](#252-multi-dimensional-optimization-based-scheduling-algorithm)
+      - [2.5.2.1 Average Per Node Allocable capacity](#2521-average-per-node-allocable-capacity)
+      - [2.5.2.2 Resource Equivalence](#2522-resource-equivalence)
+    - [2.5.3 Dynamic Geolocation and Resource Profile Based Partition Scheme](#253-dynamic-geolocation-and-resource-profile-based-partition-scheme)
+    - [2.5.4 Conflict-Aware VM/Container Request Distribution and Scheduling Mechanism](#254-conflict-aware-vmcontainer-request-distribution-and-scheduling-mechanism)
+    - [2.5.5 Smart Conflict Avoidance Approach](#255-smart-conflict-avoidance-approach)
+    - [2.5.6 Scheduling Failure](#256-scheduling-failure)
+    - [2.5.7 Scheduling of Scale Set](#257-scheduling-of-scale-set)
+    - [2.5.8 Priority Scheduling and Fair Scheduling to Avoid Security Attack](#258-priority-scheduling-and-fair-scheduling-to-avoid-security-attack)
+- [3. System Flow and Interface Design](#3-system-flow-and-interface-design)
+
+**Revision Records**
+
+|    Date    | Revision Version |              Change Description              |      Author      |
+|:----------:|:----------------:|:--------------------------------------------:|:----------------:|
+| 2020-08-25 |        1.0       | First version of Global Scheduler Design Doc | Cathy Hong Zhang |
+|            |                  |                                              |                  |
+|            |                  |                                              |                  |
+|            |                  |                                              |                  |
+|            |                  |                                              |                  |
+
+# 1. Introduction
+
+## 1.1 Problem Statement
+
+Edge Cloud brings computing resources to the edge of the network and provides secure, low latency, and high QoS computing service to mobile applications/devices at the edge and mitigates the network congestion on the Internet. Edge Cloud has several characteristics: the number of Edge Cloud Sites is large, they are all distributed, and each one has a small limited resource pool. It could happen that a resource request, e.g. a VM or container request, cannot be served by an Edge Cloud site due to resource limitation. So, resource sharing and coordination among multiple Edge Cloud Sites are needed, i.e. Edge Computing must globally support resource scheduling.
+
+Global scheduling is also needed in the Public Cloud and Hybrid Cloud. At any given time, one cloud data center may have resource strain while another cloud data center may be in light usage. Global scheduling makes better scheduling decisions and renders better resource utilization and service performance.
+
+## 1.2 Goals and Scope
```

Comment:
1、Our design constraints 1K DCs+Edge sites ,the number of site depend on how we define the edge ： edge DC or  edge device。
2、Do we need to  clarify QPS